jest.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
   },
   resolver: '@nrwl/jest/plugins/resolver',
   moduleFileExtensions: ['ts', 'js', 'html'],
-  coverageReporters: ['html'],
+  collectCoverage: true,
+  coverageReporters: ['html', 'text'],
```

Comment:
Question, what does coverageReporters do and why specify `text`? Looking at [jest docs](https://jestjs.io/docs/en/configuration#coveragereporters-arraystring--string-options) didn't help me understand this.

Reply:
Good question!

In order for the coverage report to be printed out, we need to add `text` to coverageReporters. 
I also set collectCoverage to `true` here so that it always print the coverage report without us needed to pass `--coverage` through CLI.

Reference here: https://github.com/nrwl/nx/issues/1337

Reply:
Perfect! What I thought but wasn't sure, thanks

Reply:
Np 😄